lib/net.js

```diff
@@ -338,7 +338,7 @@ function shutdownSocket(self, callback) {
 
 // the user has called .end(), and all the bytes have been
 // sent out to the other side.
-Socket.prototype._final = function(cb) {
```

Comment:
@ErnestoSalazar Congrats on your first PR in Node.js!
However, everything assigned to prototype should be good ([comment](https://github.com/nodejs/node/pull/20021#pullrequestreview-112200101)) and not required to be named.

Reply:
So, is there any reason to keep that issue open? i see only anonymous functions that linked to prototypes, thanks for your fast reply :D